services/web/server/src/simcore_service_webserver/socketio/__init__.py

```diff
@@ -19,11 +19,6 @@
 def setup(app: web.Application):
     mgr = None
     cfg = app[APP_CONFIG_KEY][CONFIG_SECTION_NAME]
-    if "message_queue" in cfg and cfg["message_queue"]:
-        mq_config = cfg["message_queue"]
-        url = f"amqp://{mq_config['user']}:{mq_config['password']}@{mq_config['host']}:{mq_config['port']}"
-        mgr = AsyncAioPikaManager(url=url, logger=log)
-
     sio = AsyncServer(async_mode="aiohttp", client_manager=mgr, logging=log)
```

Comment:
does this server shuts down correctly as is?

Reply:
that is a good question...